libs/brackets-server/embedded-ext/ConfigXmlEditor/scripts/UpdateManager.js

```diff
@@ -157,7 +157,7 @@ define(function (require, exports, module) {
 
         let selectedPrivileges = configPage.find("#privilegeSelector")[0].options;
         for (let i = 0; i < selectedPrivileges.length; i++) {
-            privilegeElement = xmlDoc.createElement("tizen:privilege");
+            privilegeElement = xmlDoc.createElementNS("http://tizen.org/ns/widgets", "tizen:privilege");
```

Comment:
Some lines seem to have http://www.w3.org/ns/widgets. but for others, they seem to have http://tizen.org/ns/widgets.
What's the rule to specifying uri as parameter ?

Reply:
If the element has the 'tizen::' namespace, it needs the 'http://tizen.org/ns/widgets'. But if the element doesn't have the 'tizen::' namespace, it needs the 'http://www.w3.org/ns/widgets'.
That is rule of xml.